quiz-masters/src/App.scss

```diff
@@ -1,33 +1,92 @@
-.logIn{
-  background-color: #D9DCD6;
+$primaryColor: #15616D;
+$secondaryColor: #D9DCD6;
+$millionairesPrimaryColor: #A393BF;
+$millionairesSecondaryColor: #672A4E;
+$jeopardyPrimaryColor: #16425B;
+$jeopardySecondaryColor: #FCA311;
+
+.primaryColor {
+  background-color: $primaryColor;
+}
+
+.secondaryColor {
+  background-color: $secondaryColor;
+}
+
+.millionairesPrimaryColor {
+  background-color: $millionairesPrimaryColor;
+}
+
+.millionairesSecondaryColor {
+  background-color: $millionairesSecondaryColor;
+}
 
-  .titleContainer{
+.jeopardyPrimaryColor {
+  background-color: $jeopardyPrimaryColor;
+}
+
+.jeopardySecondaryColor {
+  background-color: $jeopardySecondaryColor;
+}
+
+.color{
+ background-color: #037171;
+}
+
+
+.alertInvisible{
```

Comment:
sometimes you give space after name sometimes no. Please try to add linter it will simplify this for you and add space always.